### PR TITLE
Task/expand explanations

### PR DESCRIPTION
**Pull request by jusoskina**

### CODE REVIEW

File: source/includes/supply/_entry-links.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <<<<<<< HEAD
```
Comment (brenankeller) [inline source/includes/supply/_entry-links.md]: @jusoskina Looks like some merge markdown was leftover here.

File: source/includes/supply/_entry-links.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
 The Entry Links resource enables suppliers to create, update, and retrieve supplier survey entry links as well as specify [supplier link and tracking types](#get-list-global-definitions).
 =======
-The Entry Links resource enables suppliers to create, update, and retrieve supplier survey entry links as well as specify [supplier link and tracking types](#get-list-global-definitions). The entry links can then be used to send a respondent to a particular survey, and attaching important information in a query string, such as the unique panelist ID (PID), unique session ID (MID) or respondent's profiling data.
+The Entry Links resource enables suppliers to create, update, and retrieve supplier survey entry links as well as specify [supplier link and tracking types](#get-list-global-definitions). The entry links can then be used to send a respondent to a particular survey, and attaching important information in a query string, such as the unique panelist ID (PID), unique session ID (MID) or respondent's profile data.
 >>>>>>> 4c100e222079001228cb43f071048c50c14d28dc
```
Comment (brenankeller) [inline source/includes/supply/_entry-links.md]: Closed here

File: source/includes/demand/_survey-groups.md [hunk 3]
```diff
@@ -422,7 +422,11 @@ POST  https://api.samplicio.us/Demand/v1/SurveyGroups/{SurveyGroupID}
 > Example Request
 
 ```shell
+<<<<<<< HEAD
```
Comment (brenankeller) [inline source/includes/demand/_survey-groups.md]: @jusoskina Looks like another merge artifact. Double check that there are no results for "<<<<" in your source.

**Comment by brenankeller** [comment]
@jusoskina Where do we stand on this?

**Comment by jusoskina** [comment]
Resolved in browser, should be good to go!

**Pull request merged**